packages/core/src/components/Dropdown/components/MultiValueContainer/MultiValueContainer.module.scss

```diff
@@ -11,10 +11,18 @@
   --chips-margin: 2px;
 }
 
-.multiselect-chip-single-line {
+.multiselect-chip-single-line-medium {
```

Comment:
What about other sizes? I see that it supports also XS, XXS etc
![Screenshot 2024-06-25 at 14 45 33](https://github.com/mondaycom/vibe/assets/18269880/aad2809c-eb17-4aee-9ce4-09a86a340a30)

Reply:
XS & XXS don't work, we should remove them in the next major